qiskit/transpiler/passmanager.py

```diff
@@ -207,29 +214,61 @@ def _in_parallel(circuit, pm_dill=None):
         result = running_passmanager.run(circuit)
         return result
 
-    def _run_several_circuits(self, circuits):
+    def _run_several_circuits(self, circuits, output_name=None, callback=None):
         """Run all the passes on each of the circuits in the circuits list
-
-        Args:
-            circuits (list[QuantumCircuit]): circuit to transform via all the registered passes
-
         Returns:
             list[QuantumCircuit]: Transformed circuits.
         """
+        # TODO support for List(output_name) and List(callback)
+        del output_name
+        del callback
```

Comment:
why are these deleted?

Reply:
I will add support for List(output_name) and List(callback) in a coming PR. I will remove these del after that.

Reply:
can you leave a TODO comment then

Reply:
It's in line 222.